packages/components/select/src/useSelect.ts

```diff
@@ -362,7 +362,6 @@ export const useSelect = (props, states: States, ctx) => {
 
   // methods
   const resetInputHeight = () => {
-    if (props.collapseTags && !props.filterable) return
```

Comment:
I do understand why we were removing the first part of this condition, but what about the last part? Could you verify whether this will cause any regressions?

Reply:
`collapseTags && filterable`, `!collapseTags && filterable`, `collapseTags && !filterable`, `!collapseTags && !filterable`:
All of above case, tags may be line feeds. So I remove line 365.